lighthouse-core/config/default-config.js

```diff
@@ -5,9 +5,21 @@
  */
 'use strict';
 
+/* eslint-disable max-len */
+
 const constants = require('./constants');
+const i18n = require('../lib/i18n');
 
-/* eslint-disable max-len */
+const UIStrings = {
```

Comment:
FWIW @brendankenny I think these are the only strings we really need for performance section, so it's relatively small

if you want to be noodling on a much better config solution before we start on the other categories, nows a good time :)